ogusa/parameters.py

```diff
@@ -301,7 +301,6 @@ def get_parameters(test=False, baseline=False, guid='', user_modifiable=False, m
     T = int(4 * S)
     BW = int(10)
 
-    start_year = 2016
     starting_age = 20
```

Comment:
@jdebacker Will setting this to 2018 break anything?  Is there a reason why this was hard coded?

Reply:
@hdoupe This needs to be fixed.  I think hard coding comes from very early on before we were passing a `start_year` variable.  I don't think changing this to 2018 will be an issue.  

*But* what we need to do have start_year set only in one place.  Besides this hard coded value, it is currently in the `user_params` dictionary (specified in the runner script and passed into `execute.py`) but the value in `parameters.py` is not overwritten with this user defined value.  The value in the `user_params` dictionary only affects the tax functions estimation (see line 50 of `execute.py`).  We want it to also affect the demographics in the call to `dem.get_pop_objs()` in `parameters.py`.

I think the best way to do this is to make `start_year` a kwarg for `parameters.get_parameters()`.  But I'm open to other methods.